Misc/NEWS.d/next/Library/2017-10-23.bpo-31854.fh8334f.rst

```diff
@@ -0,0 +1 @@
+Add ``mmap.ACCESS_DEFAULT`` to namespace.
```

Comment:
I don't understand "Add ... to namespace": which namespace?

I propose to rephase the NEWS entry as: "Add ``mmap.ACCESS_DEFAULT`` constant."